### PR TITLE
{WIP} Added a new API

### PR DESCRIPTION
**Pull request by GauravV-02**
WIP @humphd

### CODE REVIEW

File: src/extensions/default/bramble/lib/RemoteCommandHandler.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ define(function (require, exports, module) {
     var FileSystem         = brackets.getModule("filesystem/FileSystem");
     var HTMLRewriter       = brackets.getModule("filesystem/impls/filer/lib/HTMLRewriter");
     var SidebarView        = brackets.getModule("project/SidebarView");
+    var PreviewView        = brackets.getModule("iframe-browser");
```
Comment (humphd) [inline src/extensions/default/bramble/lib/RemoteCommandHandler.js]: Modules that are defined outside an extension (i.e., Brackets "core" modules) can be loaded via `brackets.getModule()`.  However, anything that is defined by this extension (e.g., the `iframe-browser` module) must be loaded with `require()`.

Move this line down to line 29, and change it to use `require()`.

**Comment by GauravV-02** [comment]
@humphd Its working, I am able to hide/show preview pane.

**Comment by flukeout** [comment]
Hey @GauravV-02 - can I get some instructions for testing these methods locally when running brackets? Cheers!

**Comment by GauravV-02** [comment]
Hi @flukeout, For some reason I didn't got notification. Added preview pane hide/show #897
this is my new pr. You can test it on web console by doing the following steps:
1) Run the app locally and open console.
2) Change branmble... to top and test those commands
capture1
capture

**Comment by flukeout** [comment]
Thanks @GauravV-02 I'll test it out tomorrow!

**Comment by gideonthomas** [comment]
@GauravV-02, if this is an old PR and you've migrated your code to another PR, please close this one.

**Pull request closed without merging**